src/ducks/balance/components/AccountsList.jsx

```diff
@@ -18,23 +20,52 @@ class AccountsList extends React.PureComponent {
   }
 
   render() {
-    const { accounts } = this.props
+    const {
+      accounts,
+      breakpoints: { isMobile }
+    } = this.props
 
     return (
       <ol className={styles.AccountsList}>
         {sortBy(accounts, a => a.balance).map(a => (
           <li
             key={a._id}
-            className={styles.AccountsList__item}
+            className={styles.AccountsListItem}
             onClick={this.goToTransactionsFilteredByDoc(a)}
           >
-            <span>{getAccountLabel(a)}</span>
+            <span className={styles.AccountsListItem__column}>
+              {getAccountLabel(a)}
+            </span>
+            {!isMobile && (
+              <span
+                className={cx(
+                  styles.AccountsListItem__column,
+                  styles['AccountsListItem__column--secondary']
+                )}
+              >
+                N°
+                {a.number}
+              </span>
+            )}
+            {!isMobile && (
```

Comment:
this is the same condition of https://github.com/cozy/cozy-banks/pull/743/files#diff-93efe2b339535bf4f631602808c2b4d6R39 maybe you can use:

```
{!isMobile && (
  <React.Fragment>
    <span
      className={cx(
        styles.AccountsListItem__column,
        styles['AccountsListItem__column--secondary']
      )}
    >
      N°
      {a.number}
    </span>
    <span
      className={cx(
        styles.AccountsListItem__column,
        styles['AccountsListItem__column--secondary']
      )}
    >
      {a.institutionLabel}
    </span>
  </React.Fragment>
)}
```